src/modules/gamma.c

```diff
@@ -243,37 +243,38 @@ void fill_gamma_table(uint16_t *r, uint16_t *g, uint16_t *b, double br, uint32_t
     }
 }
 
-void store_gamma_brightness(const char *id, double brightness) {
+int set_gamma_brightness(const char *id, double brightness) {
+    int error;
+    gamma_client *sc = fetch_client(NULL, "", "", &error);
```

Comment:
Fetch client before storing new backlight value, because `fetch_client` internally calls `plugins->get` that is adjusted given gamma backlight set, thus leading to incorrect `sc->current_temp`.